src/components/Table/GridTable.test.tsx

```diff
@@ -712,27 +712,27 @@ describe("GridTable", () => {
           sorting={{ on: "client", initial: [1, "ASC"] }}
           rows={[
             simpleHeader,
-            // And the 1st row is pinned first
-            { kind: "data", id: "1", pin: "first", data: { name: "a", value: 11 } },
-            { kind: "data", id: "2", pin: "first", data: { name: "b", value: 10 } },
```

Comment:
I just updated this test case to put the "pin: first" rows at the bottom, and the "pin: last" rows at the top, to make extra sure they were getting proactively moved to the right spot.